### PR TITLE
Refactor runtime to webpacker JS

### PR DESCRIPTION
**Pull request by aitchiss**
## What type of PR is this? (check all applicable)

- [X] Refactor

## Description

When working on the new admin member index view, I reached for `Runtime.copyToClipboard` and realised this is only available in the 'user-facing' part of the app, loaded as part of the asset pipeline along with older initializer code.

As we make more views in the admin area responsive and mobile-friendly, I think it makes sense to move `Runtime` helpers to the webpacker JS. This allows us to re-use the code, importing only the method calls we need.

The Runtime helpers are currently used also by legacy initializers JS, and to allow the code to continue to be accessible there I've used the `window.Forem` namespace we have for this purpose. I have imported all of the Runtime helpers to this namespace, which differs from our approach with other namespaced functions. This is a conscious choice - unlike those other functions, runtime helpers don't rely on Preact, and as an import is fairly lightweight. Making this a dynamic import IMO would cause unnecessary complexity. 

## Related Tickets & Documents

Prompted by work on https://github.com/forem/forem/issues/16911

## QA Instructions, Screenshots, Recordings

We need to regression test areas that use the `Runtime` helpers, ideally on different OS and devices. I've run through the sections touched in this PR on desktop, android mobile, and iOS simulator (both in browser and in mobile app), but I'd be grateful for any more manual testing particularly if anyone could spin it up on a iOS native device.

### UI accessibility concerns?

N/A

## Added/updated tests?

- [X] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [X] I will share this change internally with the appropriate teams

## [optional] What gif best describes this PR or how it makes you feel?

moving a sofa

### CODE REVIEW

File: app/javascript/article-form/components/__tests__/ImageUploader.test.jsx [hunk 1]
```diff
@@ -13,16 +13,35 @@ import '@testing-library/jest-dom';
 
 global.fetch = fetch;
 
+const windowNavigator = window.navigator;
+const windowWebkit = window.webkit;
```
Comment (jeremyf) [inline app/javascript/article-form/components/__tests__/ImageUploader.test.jsx:17]: Nice.  This makes sense as I read through what's happening.

**Review by Ridhwana — APPROVED** [review]
Code looks good, and I've tested this on desktop.

**Comment by aitchiss** [comment]
Since this mostly impacts the mobile world it would be awesome if you could take a look too @fdocr 🙏

**Comment by fdocr** [comment]
> Since this mostly impacts the mobile world it would be awesome if you could take a look too @fdocr 🙏

Absolutely! Testing through the PR locally now, sorry for the delay I got a bit sidetracked yesterday 😅

**Pull request merged**